lightly/models/modules/masked_autoencoder_timm.py

```diff
@@ -226,46 +244,15 @@ def encode(
         # convert images to tokens and add class token if needed
         input = self.images_to_tokens(images, prepend_class_token=True)
         # add positional encoding
-        input = input + self.interpolate_pos_encoding(input)
+        input = input + self.pos_embed
```

Comment:
```suggestion
        input = self._pos_embed(input)
```

This should also take care of the interpolation for the positional embeddings. It calls `input + self.pos_embed` internally. See: https://github.com/huggingface/pytorch-image-models/blob/992976f007a1f81a1653686ce90be917f50fc0f8/timm/models/vision_transformer.py#L586

This should be a bit less hacky than the way the MAE codebase does it.

Reply:
I am not 100% convinced by this suggestion TBH. Just because the `_pos_embed` is a private method of `VisionTransformer`. WDYT?

Reply:
Removed also the `images_to_tokens` method since the addition of the classification token is also handled by `VisionTransformer`.